supervisor/docker/stats.py

```diff
@@ -45,12 +45,9 @@ def _calc_cpu_percent(self, stats):
             stats["cpu_stats"]["system_cpu_usage"]
             - stats["precpu_stats"]["system_cpu_usage"]
         )
-        online_cpu = stats["cpu_stats"]["online_cpus"]
 
-        if online_cpu == 0.0:
-            online_cpu = len(stats["cpu_stats"]["cpu_usage"]["percpu_usage"])
         if system_delta > 0.0 and cpu_delta > 0.0:
-            self._cpu = (cpu_delta / system_delta) * online_cpu * 100.0
+            self._cpu = (cpu_delta / system_delta) * 100.0
```

Comment:
should we add an else self._cpu = 0.0?

Reply:
in what case would that be needed?

Reply:
I guess if everything is 0 :shrug: 
Added guard and updated test